frappe/website/js/web_form.js

```diff
@@ -281,14 +281,10 @@ frappe.ready(function() {
 			callback: function(data) {
 				if(!data.exc) {
 					frappe.doc_name = data.message;
-					if(!frappe.login_required) {
```

Comment:
`login_required` is a flag whether login is necessary or not, not a status whether a user is logged in or not. What is the use case for this?